aether-ui/aether/ui/assets/apps/pipeline/PipelineList.jsx

```diff
@@ -59,19 +123,24 @@ class PipelineList extends Component {
                 key={pipeline.id}
                 pipeline={pipeline}
                 history={this.props.history}
+                rename={() => {}}
```

Comment:
:roll_eyes:

Reply:
why not `rename={() => { window.alert('we are working hard on this... or maybe we forgot to do it') }}`? :wink:

Reply:
Actually, I would hide this till it's  ready...